packages/typespec-python/src/emitter.ts

```diff
@@ -214,14 +214,16 @@ function getEffectiveSchemaType(context: SdkContext, type: Model): Model {
     return type;
 }
 
-function getEntityType(context: SdkContext, entity: ModelProperty): any {
-    const result = getType(context, entity.type);
+function emitModelPropertyType(context: SdkContext, entity: ModelProperty): Record<string, any> {
+    const result = emitType(context, entity.type);
```

Comment:
Why do you change `getType` to `emitType` here?

Reply:
`emitModelPropertyType` could be considered a new function and has a different logic, and `getEntityType` has been removed.